updating links (#210)
rstudio/education.rstudio.com: content/teach/materials/index.md

# Courses

> Most teachers enjoy developing their own instruction materials, but the need for exercises, homework, exams, slides, and other supporting materials make this a big job. Below are some open source materials developed at Posit and elsewhere that you can freely adapt and use for your R teaching.

Reviewer: ```suggestion
Most teachers enjoy developing their own instruction materials, but the need for exercises, homework, exams, slides, and other supporting materials makes this a big job. Below are some open-source materials developed at Posit and elsewhere that you can freely adapt and use for your R teaching.
```

- **Get a complete [data science course in a box](https://datasciencebox.org).** [Data Science in a Box](https://datasciencebox.org) contains the complete materials for teaching a semester-long introductory data science course. The "box" contains materials for an undergraduate level introductory data science course, such as slide decks, homework assignments, guided labs, sample exams, a final project assignment, as well as materials for instructors such as pedagogical tips, information on computing infrastructure, technology stack, and course logistics. The [website](https://datasciencebox.org) exposes the source materials that live in a [GitHub repository](https://github.com/rstudio-education/datascience-box) and use datasets from the [dsbox package](https://github.com/rstudio-education/dsbox).

- **Teach with [STAT 545](https://stat545.com/).**

> STAT 545 is a course in data wrangling, analysis, and exploration in R with RStudio. Although the course was designed as a graduate-level, semester-long introduction to data science by [Dr. Jennifer (Jenny) Bryan](https://jennybryan.org/), the free online materials Jenny developed have been a valuable resource for self-directed learners and other educators. In Jenny's own words, STAT 545 was designed to teach _"everything that comes up during data analysis **except for statistical modelling and inference**."_ The education team at Posit has recently ported the original materials into a modern and more maintainable [bookdown website](https://stat545.com/).

Reviewer: ```suggestion
STAT 545 is a course in data wrangling, analysis, and exploration in R with RStudio. Although the course was designed as a graduate-level, semester-long introduction to data science by [Dr. Jennifer (Jenny) Bryan](https://jennybryan.org/), the free online materials Jenny developed have been a valuable resource for self-directed learners and other educators. In Jenny's own words, STAT 545 was designed to teach _"everything that comes up during data analysis **except for statistical modelling and inference**."_ We have ported the original materials to a [bookdown website](https://stat545.com/).
```

- **Use [_R for Data Science_](https://r4ds.had.co.nz/) in the classroom.** Many educators use the free online book [R for Data Science](https://r4ds.had.co.nz/) as a course textbook. There is also an [R for Data Science Instructor's Guide](https://github.com/rstudio-education/r4ds-instructors), which contains notes for people teaching [R for Data Science](https://r4ds.had.co.nz/) with each chapter's learning objectives and key points. The "unofficial" [R4DS Solutions Manual](https://jrnold.github.io/r4ds-exercise-solutions/), a community resource developed by and for educators, is also helpful if you want to use R4DS in the classroom.

# Workshops

> Many of you may have taken workshops from the Posit team, but did you know that all of RStudio's workshop materials are available for you to use in your own workshops? Below are a few of the more popular workshop respositories proven popular with teachers:

Reviewer: ```suggestion
Many of you may have taken workshops from the Posit team, but did you know that all of Posit's workshop materials are available for you to use in your own workshops? Below are a few of the more popular workshop repositories proven popular with teachers:
```

- **Teach R Markdown.** If you want to teach R Markdown, we have designed several workshops for teaching the basics to the more advanced topics. You can find the materials for a full two-day workshop on [*Advanced R Markdown*](https://arm.rbind.io/) ([source](https://github.com/rstudio-education/arm-workshop-rsc2019)), a four-hour introductory workshop on [*R Markdown for Medicine*](https://rmd4medicine.netlify.app/) ([source](https://github.com/rstudio-education/rmd4medicine)) aimed at clinical researchers, and a full-day workshop on [*Communicating with R Markdown*](https://ysc-rmarkdown.netlify.app/) ([source](https://github.com/rstudio-education/communicate-rmd-workshop)).

- **Teach Quarto.** [Quarto](https://quarto.org/) is the next generation of R Markdown, bringing reproducible publishing to Python, R, Julia, and Observable. The [Get Started with Quarto](https://quarto.org/docs/get-started/) tutorials provide step-by-step guides for multiple development environments including RStudio, VS Code, and Jupyter. Explore the [Quarto gallery](https://quarto.org/docs/gallery/) to see examples of books, websites, presentations, and dashboards you can create with your students.

> - [**Teach everything else!**](https://github.com/rstudio-education/) At the [RStudio Education GitHub Organization](https://github.com/rstudio-education/) instructors can find the materials for all workshops taught by the Posit Education team. All of these workshop materials are openly-licensed and freely-available for reuse. Please follow the reuse guidelines outlined in the licenses in the specific repositories, and enjoy leveraging quality teaching materials designed and developed by our team!

Reviewer: ```suggestion
- [**Teach everything else!**](https://github.com/rstudio-education/) Materials from the most recent Posit (and RStudio) conferences can be found at the following links: [posit::conf(2025) workshops](https://github.com/posit-conf-2025), [posit::conf(2024) workshops](https://github.com/posit-conf-2024), [posit::conf(2023) workshops](https://github.com/posit-conf-2023),
[rstudio::conf(2022) workshops](https://github.com/rstudio-conf-2022). All of these workshop materials are openly licensed and freely available for reuse. Please follow the reuse guidelines in each repository's license, and enjoy using the high-quality teaching materials our team and other workshop instructors have developed.
```

> - [**Teach everything else!**](https://github.com/rstudio-education/) At the [RStudio Education GitHub Organization](https://github.com/rstudio-education/) instructors can find the materials for all workshops taught by the Posit Education team. All of these workshop materials are openly-licensed and freely-available for reuse. Please follow the reuse guidelines outlined in the licenses in the specific repositories, and enjoy leveraging quality teaching materials designed and developed by our team!

Reviewer: For Teach everything else, I think it might make sense to link to workshop orgs for some of the most recent workshops, e.g., https://github.com/posit-conf-2025, https://github.com/posit-conf-2024, https://github.com/posit-conf-2023, 
https://github.com/rstudio-conf-2022.
